lesson_12/adrian_filimon/homework_12_af.py

```diff
@@ -0,0 +1,111 @@
+from collections.abc import MutableSequence
+
+
+class CrayonsBox(MutableSequence):
+
+    def __init__(self, iterable):
+        self._crayons = list(iterable)
+
+    def __len__(self):
+        return len(self._crayons)
+
+    def __getitem__(self, index):
+        return self._crayons[index]
+
+    ##############################################
+
+    def __setitem__(self, index, value):
+        self._crayons[index] = value
+
+    def __delitem__(self, index):
+        del self._crayons[index]
+
+    def insert(self, index, item):
```

Comment:
__setitem__, __delitem__ and insert were the minimum required to get the job done; It's cool that you've added some extra ones